Tests/Particles/NeighborParticles/MDParticleContainer.cpp

```diff
@@ -159,41 +161,40 @@ std::pair<Real, Real> MDParticleContainer::minAndMaxDistance()
 
         auto& ptile = plev[index];
         auto& aos   = ptile.GetArrayOfStructs();
-        const size_t np = aos.numParticles();
 
         auto nbor_data = m_neighbor_list[lev][index].data();
         ParticleType* pstruct = aos().dataPtr();
 
-        Gpu::DeviceScalar<Real> min_d_gpu(min_d);
-        Gpu::DeviceScalar<Real> max_d_gpu(max_d);
-
-        Real* pmin_d = min_d_gpu.dataPtr();
-        Real* pmax_d = max_d_gpu.dataPtr();
-
-        AMREX_FOR_1D ( np, i,
-        {
-            ParticleType& p1 = pstruct[i];
+        ParticleReal min_start = std::numeric_limits<ParticleReal>::max();
+        ParticleReal max_start = std::numeric_limits<ParticleReal>::min();
```

Comment:
```suggestion
        ParticleReal max_start = std::numeric_limits<ParticleReal>::lowest();
```
min is a small positive number.